opensearchpy/_async/client/ingest.py

```diff
@@ -25,42 +25,50 @@
 #  under the License.
 
 
+# ----------------------------------------------------
+# THIS CODE IS GENERATED AND MANUAL EDITS WILL BE LOST.
+#
+# To contribute, kindly make essential modifications through either the "opensearch-py client generator":
+# https://github.com/opensearch-project/opensearch-py/blob/main/utils/generate-api.py
+# or the "OpenSearch API specification" available at:
+# https://github.com/opensearch-project/opensearch-api-specification/blob/main/OpenSearch.openapi.json
+# -----------------------------------------------------
+
+
 from .utils import SKIP_IN_PATH, NamespacedClient, _make_path, query_params
 
 
 class IngestClient(NamespacedClient):
-    @query_params("master_timeout", "cluster_manager_timeout", "summary")
+    @query_params("cluster_manager_timeout", "master_timeout")
```

Comment:
I imagine this is correct?

Reply:
https://github.com/opensearch-project/opensearch-py/pull/513#issue-1916016963